base_module_remove_batch/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Module Remove Batch",
+    "summary": """
+        Add an option to remove modules in batch.
+    """,
+    "version": "12.0.2.1.0",
+    "license": "LGPL-3",
+    "category": "Base",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "data/ir_action_data.xml",
```

Comment:
I'm not sure if data such as a server action should be placed in a `data` folder. `data` folder is often used for master initialization data. Maybe a server action should be treated like other structures (views, menus, client actions,...) that are placed under `views` folder instead.